package.json

```
@@ -9,29 +9,33 @@
   },
   "scripts": {
     "start": "npm run build && fastify start -l info dist/app.js",
-    "build": "tsc",
+    "build": "rm -rf ./dist && tsc",
```

```suggestion
    "build": "rm -rf ./dist && tsc",
```

This won't work for Windows users using cmd or PowerShell.
You may need to either add a script to `scripts/` that cleans up dist and call that, or explicitly state in documentation for users to use Git Bash which comes with Git for Windows.

---

Or just don't use windows 🤣🤭